src/test/extensions/nodejsMigrationHelper.spec.ts

```diff
@@ -0,0 +1,122 @@
+import * as _ from "lodash";
+import { expect } from "chai";
+import * as sinon from "sinon";
+
+import * as nodejsMigrationHelper from "../../extensions/nodejsMigrationHelper";
+import * as prompt from "../../prompt";
+
+const NO_RUNTIME_SPEC = {
+  name: "test",
+  displayName: "Old",
+  description: "descriptive",
+  version: "1.0.0",
+  license: "MIT",
+  resources: [
+    {
+      name: "resource1",
+      type: "firebaseextensions.v1beta.function",
+      description: "desc",
+      properties: {},
+    },
+  ],
+  author: { authorName: "Tester" },
+  contributors: [{ authorName: "Tester 2" }],
+  billingRequired: true,
+  sourceUrl: "test.com",
+  params: [],
+};
+
+const NODE8_SPEC = {
+  name: "test",
+  displayName: "Old",
+  description: "descriptive",
+  version: "1.0.0",
+  license: "MIT",
+  resources: [
+    {
+      name: "resource1",
+      type: "firebaseextensions.v1beta.function",
+      description: "desc",
+      properties: { runtime: "nodejs8" },
+    },
+  ],
+  author: { authorName: "Tester" },
+  contributors: [{ authorName: "Tester 2" }],
+  billingRequired: true,
+  sourceUrl: "test.com",
+  params: [],
+};
+
+const NODE10_SPEC = {
+  name: "test",
+  displayName: "Old",
+  description: "descriptive",
+  version: "1.0.0",
+  license: "MIT",
+  resources: [
+    {
+      name: "resource1",
+      type: "firebaseextensions.v1beta.function",
+      description: "desc",
+      properties: { runtime: "nodejs10" },
+    },
+  ],
+  author: { authorName: "Tester" },
+  contributors: [{ authorName: "Tester 2" }],
+  billingRequired: true,
+  sourceUrl: "test.com",
+  params: [],
+};
+
+describe("nodejsMigrationHelper", () => {
```

Comment:
Optional: Add a test case for if the user says no to the prompt

Reply:
Done